docs/tutorial/using-checkbox/installing_checkbox.rst

```diff
@@ -8,7 +8,7 @@ In order to install Checkbox you are going to need two parts, a ``runtime``
 and a ``frontend``.
 
 First we need to install the ``runtime``, this tutorial will use the ``checkbox22``
-runtime on Ubuntu22. We also offer versions for ``16``, ``18`` and ``20``
+runtime on Ubuntu22. We also offer versions for ``16``, ``18`` and ``20``.
```

Comment:
I suppose this tutorial is to be followed on a classic desktop version of Ubuntu, in which case you should use "Ubuntu 22.04". (it's either that, either "Ubuntu Core 22" but I think it's easier to use desktop version of Ubuntu)